src/js/health-beta/containers/HealthBetaEnrollment.jsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import RequiredLoginView from '../../common/components/RequiredLoginView';
+import { registerBeta } from '../actions';
+
+class HealthBetaEnrollment extends React.Component {
+
+  componentWillMount() {
```

Comment:
Same comment as before about using `componentDidMount` instead for invoking API requests.